Flank/flank
# feat: Load test cases durations from previous run and use for sharding

Fixes #1988 

## Test Plan

1. Run `flank corellium test android run -c="./test_configs/flank-corellium.yml"` several times.
2. Observe the generated `android-shards.json` have durations different than the default (120)

## Drawbacks

The first test case in a suite always has a duration increased by initialization overhead. This is a `am instrument` command behavior, that not depends on flank implementation. As result, the sharding calculations can be more imprecise for cases with a small number of quick test cases with duration less than initialization overhead. 

A possible solution could be evaluating overhead time based on previous JUnit results, and taking them into account in sharding calculations.

## Checklist
